### DISCUSSION TITLE
idea cannot log in copilot

### DISCUSSION THREAD

**Discussion by joeylin** [community/community#41887] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/2030833/208020725-4b26fefa-436a-4215-943d-03fbd4df1a21.png)

我在github上点击授权之后，idea一直没有反应，copilot状态也是未登录的状态

idea版本：
- Runtime version: 17.0.5+7-b469.71 x86_64
- VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

copilot版本：1.1.37.2188
操作系统：macos big sur 11.7

尝试过使用系统代理，也一直没效果。代理用其他工具测试是OK的

**Comment by iamcjwilliams** [comment]
@joeylin Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-a-jetbrains-ide

**Comment by Akash1134** [comment]
Hi @joeylin It seems that you're running into an issue that users have reported in the past, sorry that you have it. Please check the solution for this [discussion](https://github.com/community/community/discussions/8333) and let us know if that helps. Thanks!
